clients/cobol-lsp-vscode-extension/src/commands/ClearCopybookCacheCommand.ts

```diff
@@ -16,27 +16,26 @@ import * as path from "node:path";
 import * as vscode from "vscode";
 import { FileType } from "vscode";
 import {
-  C4Z_FOLDER,
   CLEARING_COPYBOOK_CACHE,
   COPYBOOK_CACHE_CLEARED_INFO,
   COPYBOOKS_FOLDER,
+  ZOWE_FOLDER,
 } from "../constants";
+import { Utils } from "../services/util/Utils";
 
 /**
- * Clears the downloaded copybook cache folder ({workspace}/.c4z/.copybooks).
+ * Clears the downloaded copybook cache folder ({globalStoragePath}/.zowe/.copybooks).
  *
  */
-export function clearCache() {
+export function clearCache(uri: vscode.Uri) {
   vscode.window.setStatusBarMessage(
     CLEARING_COPYBOOK_CACHE,
     Promise.resolve().then(
       () => {
-        const firstWorkspaceFolder = vscode.workspace.workspaceFolders![0];
-        const folderUri = firstWorkspaceFolder!.uri;
-        const fileUri = folderUri.with({
-          path: path.join(folderUri.fsPath, C4Z_FOLDER, COPYBOOKS_FOLDER),
+        const zoweFileUri = uri.with({
+          path: path.join(uri.fsPath, ZOWE_FOLDER, COPYBOOKS_FOLDER),
```

Comment:
Do I guess correctly that the `vscode.Uri.joinPath` is not available in tests?

Reply:
this was just an old code used as it is. 

joinPath would be better but eventually both do same things.